oam/install_scripts/columnstore-post-install.in

```diff
@@ -300,6 +297,14 @@ if [ ! -z "$MCS_USE_S3_STORAGE" ] && [ $MCS_USE_S3_STORAGE -eq 1 ]; then
   fi  
 fi
 
+#change ownership/permissions to be able to run columnstore as non-root
+chown -R mysql:mysql /var/log/mariadb/columnstore
+chown -R mysql:mysql /etc/columnstore
+chown -R mysql:mysql /var/lib/columnstore
+chown -R mysql:mysql /tmp/columnstore_tmp_files
+chmod 777 /tmp/columnstore_tmp_files
+chmod 777 /dev/shm
```

Comment:
This is an ugly change.

Reply:
It's from our documentation for unprivileged-user-installation.